recipes/cfm/meta.yaml

```diff
@@ -1,26 +1,25 @@
+{% set name = "cfm" %}
+{% set version = "2.4.1" %}
```

Comment:
Question: Do you think that 33 (which seems to be the release number of the sourceforge code) corresponds to 2.4(.1)? 

Problem: I guess we can't use version 2.4.1 here, since 33>2.4.1. Not sure what is the best solution for this, 

What is the reason not to update to the latest version 4.x (https://bitbucket.org/wishartlab/cfm-id-code/downloads/?tab=tags)?

Reply:
Maybe we need to leave this recipe alone .. maybe deprecate it (if such a thing is possible) .. and start a new `cfm-id` recipe.

Reply:
Yes - I like your suggestion. cfm-id seems to be the name used as an umbrella for all the tools included.